demo/src/main/java/com/example/demo/contoller/CalcController.java

```diff
@@ -0,0 +1,15 @@
+package com.example.demo.contoller;
+
+import com.example.demo.dto.CalculatorDTO;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/calculator")
+public class CalcController {
+	  
+	   @PostMapping(value = "/sum")
+	   @ResponseBody
+	   public Integer addValue(@RequestBody CalculatorDTO calculatorDTO) {
+			 return calculatorDTO.getFirstValue() + calculatorDTO.getSecondValue();
```

Comment:
Why is Controller invoking a DTO class for fulfilling the request? Remember, DTO classes are passive elements, they can't be active elements sending or receiving requests.

Reply:
DTO is used to hide some column from request body.